lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -882,6 +889,102 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
+        w <- fixtureWallet ctx
+        let amt = minUTxOValue (_mainEra ctx)
+
+        -- Construct tx
+        payload <- mkTxPayload ctx w amt
+        let constructEndpoint = Link.createUnsignedTransaction @'Shelley w
+        sealedTx <- getFromResponse (#transaction . #getApiT . #serialisedTx) <$>
+            request @(ApiConstructTransaction n) ctx constructEndpoint Default payload
+
+        -- Submit tx
+        let submitEndpoint = Link.postExternalTransaction
+        let headers = Headers
+                [ ("Content-Type", "application/octet-stream")
+                , ("Accept", "application/json")
+                ]
+        r <- request @ApiTxId ctx submitEndpoint headers (NonJson $ BL.fromStrict sealedTx)
+        verify r
+            [ expectResponseCode HTTP.status500
+            ]
+
+    it "TRANS_NEW_SIGN_03 - Accepts signature for withdrawals" $ \ctx -> runResourceT $ do
+        (w, _) <- rewardWallet ctx
+
+        -- Construct tx
+        let payload = Json [json|{"withdrawal": "self"}|]
+        let constructEndpoint = Link.createUnsignedTransaction @'Shelley w
+        (_, apiTx) <- unsafeRequest @(ApiConstructTransaction n) ctx constructEndpoint payload
+        length (withdrawals $ coinSelection apiTx) `shouldBe` 1
+
+        -- Sign tx
+        let sealedTx = serialisedTx . getApiT . transaction $ apiTx
+            toSign = Json [json|
+                { "transaction": #{sealedTx}
+                , "passphrase": #{fixturePassphrase}
+                }|]
+        let signEndpoint = Link.signTransaction @'Shelley w
+        rSignedTx <- request @ApiSignedTransaction ctx signEndpoint Default toSign
+        verify rSignedTx
+            [ expectResponseCode HTTP.status202
+            ]
+        let signedTx = getFromResponse (#transaction . #getApiT . #serialisedTx) rSignedTx
+
+        -- Currently fails to deserialise:
+        --
+        -- 84a70081825820410a9cd4af08b3abe25c2d3b87af4c23d0bb2fb7577b639d5cfbdfe13a4a696c0c0d80018182583901059f0c7b9899793d2c9afaeff4fd09bedd9df3b8cb1b9c301ab8e0f7fb3c13a29d3798f1b77b47f2ddb31c19326b87ed6f71fb9a27133ad51b000001001d19d714021a000220ec03198d0f05a1581de1fb3c13a29d3798f1b77b47f2ddb31c19326b87ed6f71fb9a27133ad51b000000e8d4a510000e80a0f5f6
+
+        -- Submit tx
+        let submitEndpoint = Link.postExternalTransaction
+        let headers = Headers
+                [ ("Content-Type", "application/octet-stream")
+                , ("Accept", "application/json")
+                ]
+        r <- request @ApiTxId ctx submitEndpoint headers (NonJson $ BL.fromStrict signedTx)
+        verify r
+            [ expectResponseCode HTTP.status202
+            ]
+
+    -- TODO: Moar tests scenarios to cover in the context of sign-transactions
+    --
+    -- - Signing with non-own / scripts inputs
+    -- - Signing with pre-existing witnesses
+    -- - Signing with collaterals
+    -- - Signing with extra required signatures
+    -- - ideas?
```

Comment:
There are actually several cases in this file that were created only up to "Construct" that, I guess, can be now filled with sign and submit steps. They all have relevant comment:
```
TODO: now we should sign it and send it in two steps
```
or
```
TODO: sign
```